### DISCUSSION TITLE
WikiLinks: Allow them to be saved as conventional links

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#794] [Feature Requests]
GitJournal now supports wiki style links `[[word]]`. This is not part of the Markdown syntax, perhaps it would be useful to provide an option to automatically convert wiki style links to the conventional syntax on save.

This - of course - would be configurable.

---
> Please vote on this issue with a 👍reaction. It's one of the primary mechanisms used to prioritize issues. Please also subscribe to the issue if you're interested, as that way I can discuss possible implementations and share mockups.

**Comment by necopinus** [comment]
Would be great if this was compatible with Foam’s approach!
